packages/react-kit/src/components/DateInput/DateInput.model.ts

```diff
@@ -100,5 +106,7 @@ export function incrementMonth(month: number): number {
 }
 
 export function incrementMonthOption(month: Option<number>): Option<number> {
-	return month.map(incrementMonth).alt(some(0));
+	return pipe(month, map(incrementMonth), alt(constant(some(0))));
 }
+
+export const inc = (value: number): number => value + 1;
```

Comment:
This function looks like common helper, that doesn't relate to date input model.

Reply:
It was moved here from the component. Anyway it is not related to fp-ts@2.0 migration